src/core/Akka/Configuration/Hocon/HoconTokenizer.cs

```diff
@@ -602,6 +636,34 @@ public bool IsSubstitutionStart()
             return Matches("${");
         }
 
+        public bool IsInclude()
+        {
+            Push();
+            try
+            {
+                if (Matches("include"))
+                {
+                    Take("include".Length);
+
+                    if (IsWhitespaceOrComment())
+                    {
+                        PullWhitespaceAndComments();
+
+                        var rest = PullUnquotedKey();
+                        if (rest.Value.StartsWith("'") && rest.Value.EndsWith("'"))
+                        {
+                            return true;
+                        }
+                    }
+                }
+                return false;
+            }
+            finally
+            {
+                Pop();
```

Comment:
This is only looking/peeking at the current pos, not consuming, thus we _always_ rollback the parser position.